docker-compose.yml

```diff
@@ -14,19 +14,22 @@ services:
     ports:
       - "7777:7777"
     environment:
-      - RUN_DEV_SERVER=True
-      - PORT=7777
+      - RUN_DEV_SERVER=False
```

Comment:
I sorta meant that this (and really this entire docker-compose file) should be the defaults for the dev environment. This docker-compose file doesn't affect production or staging deployments at all (ops sets these environment variables in their puppet configurations, they don't use docker-compose). "Secure by default" comes into play with the default click options in `app.py` not in this file.

Considering all this, I think it makes more sense for this to default to True and so on with the rest of these options (they should be defaults for the developer environment).